tajo-common/src/main/proto/DataTypes.proto

```diff
@@ -114,3 +114,7 @@ message DataType {
    */
   optional int32 num_nested_fields = 4;
 }
+
+enum CodecType {
```

Comment:
DataTypes.proto seems to be not proper for CodecType, which is used for client communication.